api/Element.json

```diff
@@ -6653,7 +6658,8 @@
             ],
             "webview_android": [
               {
-                "version_added": "37"
+                "version_added": "37",
+                "notes": "From version 92 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Comment:
```suggestion
                "notes": "From version 37, returns a promise instead of <code>undefined</code>. The behavior reflects <a href='https://github.com/w3c/pointerlock/pull/49'>a proposed specification change</a>."
```